cmd/habitat/community/consensus/cluster/cluster.go

```diff
@@ -61,36 +59,23 @@ func (cm *ClusterManager) Start(proxyRules *proxy.RuleSet) error {
 		return err
 	}
 
-	// Restart any existing communities
```

Comment:
why was this removed?

Reply:
this was refactored into the community manager